crypto/pem_cert_pool.go

```diff
@@ -0,0 +1,70 @@
+package crypto
+
+import (
+	"encoding/pem"
+
+	"github.com/golang/glog"
+	"github.com/google/certificate-transparency/go/x509"
```

Comment:
Yeah it's a fair cop. But you can't mix the CT and golang x509 / asn1 / pkix packages easily. I don't want to move stuff until I've landed the next few PRs that modify the same code.